database/migrations/2019_06_11_220005_convert_existing_work_env_and_culture_to_job_poster.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class ConvertExistingWorkEnvAndCultureToJobPoster extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $jobs = DB::table('job_posters')->get();
+        $locales = ['en', 'fr'];
+        foreach ($jobs as $job) {
+            $workEnv = DB::table('work_environments')->where('manager_id', $job->manager_id)->first();
+            if ($workEnv) {
+                // Add non-localized work env fields to job
+                DB::table('job_posters')->where('id', $job->id)->update([
+                    'telework_allowed_frequency_id' => $workEnv->telework_allowed_frequency_id,
+                    'flexible_hours_frequency_id' => $workEnv->flexible_hours_frequency_id
+                ]);
+                // Add translated work env fields to job translations
+                foreach ($locales as $locale) {
+                    $workEnvTrans = DB::table('work_environment_translations')->where('work_environment_id', $workEnv->id)->where('locale', $locale)->first();
+                    if ($workEnvTrans) {
+                        DB::table('job_poster_translations')->updateOrInsert(
+                            ['job_poster_id' => $job->id, 'locale' => $locale],
+                            ['work_env_description' => $workEnvTrans->things_to_know]
+                        );
+                    }
+                }
+            }
+            $culture = DB::table('team_cultures')->where('manager_id', $job->manager_id)->first();
```

Comment:
Just need to add a line in here somewhere to populate the `culture_summary` field (if applicable)

Reply:
Fixed in d5f3d45ee726b00494c204a3f1dc72bcbb0e5e80